tss-esapi/tests/integration_tests/common/mod.rs

```diff
@@ -47,7 +47,7 @@ pub const HASH: [u8; 64] = [
 ];
 
 #[allow(dead_code)]
-pub const KEY: [u8; 512] = [
+pub const KEY: [u8; 256] = [
```

Comment:
Just a note - I've changed these because using them in the new tests below resulted in the TPM getting angry, because the key itself was not 512 bytes long, and now we're using `KEY` as the full key (via the new buffer implementation, `PrivateKeyRsa`)